src/Tasks/SynTree/Config.hs

```diff
@@ -79,9 +80,18 @@ checkSynTreeConfig SynTreeConfig {..}
     | minNodes < atLeastOccurring * 2 - 1 = reject $ do
         english "Your minimum number of nodes does not permit enough leaves for all desired literals."
         german "Minimale Anzahl der Blätter ist zu niedrig um alle Literale zu verwenden."
+    | minDepth < 1 = reject $ do
+        english "Minimal depth must be positive"
+        german "Minimale Tiefe muss positiv sein."
+    | maxNodes < minDepth = reject $ do
```

Comment:
I think that an even stronger rejection makes sense: `minNodes < minDepth` should be rejected, because it makes no sense to want depth `5` but not also at least `5` nodes.